### ISSUE TITLE
apps_dns does not find cf-app-sd-client-tls secret credential

### ISSUE THREAD

**Issue by univ0298** [cloudfoundry-incubator/kubecf#1666]
**Describe the bug**
This code from @f0rmiga https://github.com/cloudfoundry-incubator/kubecf/blob/master/chart/templates/apps_dns.yaml#L122 specifies that the cf-apps-dns deployment should be looking for a secret with name `var-cf-app-sd-client-tls` . However this feature from @jandubois (https://github.com/cloudfoundry-incubator/kubecf/commit/91e54d7cca320755ddc07a2ac9d5db1b3a21ccc7) means that whenever we specify a value for the cf-app-sd-client-tls ca/certificate/private_key , we end up with the secret being created called `cred-cf-app-sd-client-tls` . We are specifying it with these sections in our values manifest for the helm install

```
credentials:
  cf_app_sd_client_tls.ca: REDACTED
  cf_app_sd_client_tls.certificate: REDACTED
  cf_app_sd_client_tls.private_key
```

**Notice the CRED- prefix instead of the VAR- prefix that the deployment is looking for.** 

Since the VAR- prefixed one doesn't exist, the cf-apps-dns pod won't ever start due to being missing its secret.

Here is the code where @f0rmiga introduced this dependency on the VAR- variable name :

https://github.com/cloudfoundry-incubator/kubecf/commit/329c85bd5a73081404bc6ba2366ced3ceabf5f69#diff-fe38e7f7b88dae1e77e30d919760da6822842ade7353ae51ac5d127881121beaR120

Are we incorrect to specify the secret in this way? I suspect it may not be possible to have this secret specified as a var even,

**To Reproduce**
As explained above

**Expected behavior**
As explained above

**Environment**
cf-operator 7.06
kubecf 2.7.1

**Additional context**

**Comment by jandubois** [comment]
@univ0298 Please confirm that #1671 fixes your issue!

**Comment by jandubois** [comment]
> Since the VAR- prefixed one doesn't exist, the cf-apps-dns pod won't ever start due to being missing its secret.

This part I actually don't understand. The operator-generated variable should always exist; it is part of the deployment manifest: https://github.com/cloudfoundry/cf-deployment/blob/v13.17.0/cf-deployment.yml#L1839-L1845

The `credentials` section should provides overrides under a different secret name, but I thought that Quarks would always generate the default values; they would just not be used.

Anyways, maybe I misunderstand how this is implemented in Quarks; #1671 should still fix the issue either way.
